internal/serve/serve.go

```diff
@@ -362,6 +363,9 @@ func handleHTTP(o ServeOptions) *chi.Mux {
 
 			r.With(middleware.AnyRoleMiddleware(authManager, data.GetAllRoles()...)).
 				Get("/", profileHandler.GetOrganizationInfo)
+
+			r.With(middleware.AnyRoleMiddleware(authManager, data.GetAllRoles()...)).
+				Get("/logo", profileHandler.GetOrganizationLogo)
```

Comment:
Can you also update the Test_handleHTTP_unauthenticatedEndpoints and Test_handleHTTP_authenticatedEndpoints tests accordingly to reflect this change?

Reply:
Addressed in [d6af6f1](https://github.com/stellar/stellar-disbursement-platform-backend/pull/214/commits/d6af6f156af0be9a6a65b8dd72a75e3bbf80b69a)